phpseclib/Net/SSH2.php

```diff
@@ -922,6 +922,21 @@ class SSH2
      */
     private bool $smartMFA = true;
 
+    /**
+     * How many channels are currently opened
+     *
+     * @var int
+     */
+    private $channelCount = 0;
+
+    /**
+     * Does the server support multiple channels? If not then error out
+     * when multiple channels are attempted to be opened
+     *
+     * @var bool
+     */
+    private $errorOnMultipleChannels;
```

Comment:
Consider exposing a setter for this as well, such that callers can enable the feature or forcibly disable if they know the OpenSSH version match to be a non-issue due to applied workarounds.